_inc/lib/debugger/jetpack-debugger-site-health.js

```diff
@@ -5,7 +5,7 @@
 
 jQuery( document ).ready( function( $ ) {
 	var JetpackSync = {
-		inProgress: false,
+		inProgress: true,
```

Comment:
Note: 
true state is required for small sites where the first check returns "done"